src/widgets/EstimatedCasesChartInner.tsx

```diff
@@ -139,7 +139,7 @@ export const EstimatedCasesChartInner = React.memo(
                     width={50}
                   />
                   <Tooltip
-                    active={false}
+                    active={true}
```

Comment:
I see you have switched to active true a lot in this PR - what is the behavior change?

Reply:
As far as I can tell, the user-observable behavior for our application remains the same. Previously, the tooltip side effects worked worked while `active` is set to false, now it doesn't anymore.